server/src/models/configModel.ts

```diff
@@ -1,5 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+export type PostCallSurveyType = 'none' | 'msforms' | 'onequestion' | 'thirdparty';
```

Comment:
Added 'none' option here since mocking without a postCall object (as per tech doc) threw errors.
![image](https://user-images.githubusercontent.com/100614160/182998398-cf9ee99f-9267-46d5-a438-e5801c6602a2.png)

Reply:
This has been reverrted and 'none' will not b an option in current implementation